feat: criar classes base do projeto

Brschang opened the pull request.

Commit:
feat: criar classes base

LuccasArruda commented on src/Core/Entities/Ambiente.cs:
```diff
@@ -0,0 +1,14 @@
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deméter
```
# ℹ️ Requisição de Mudança
> Seria interessante se o namespace do projeto fosse escrito sem acento.

- [x] Alterado

LuccasArruda commented on src/Core/Entities/Ambiente_Pessoal.cs:
```diff
@@ -0,0 +1,13 @@
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deméter
```
# ℹ️ Requisição de Mudança
> Seria interessante se o namespace do projeto fosse escrito sem acento.

- [x] Alterado

LuccasArruda commented on src/Core/Entities/Ambiente_Profissional.cs:
```diff
@@ -0,0 +1,13 @@
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deméter
```
# ℹ️ Requisição de Mudança
> Seria interessante se o namespace do projeto fosse escrito sem acento.

- [x] Alterado

LuccasArruda commented on src/Core/Entities/Ambiente_Profissional.cs:
```diff
@@ -0,0 +1,13 @@
+{
+    internal class Ambiente_Profissional: Ambiente
+    {
+        public List<Rede_Eletrica> RedeEletrica { get; private set; }
```
# ℹ️ Requisição de Mudança
> O padrão de nomenclatura do C# para classes é o ``PascalCase``, ou seja, a primeira letra de cada palavra deve ser em caps-lock e as demais, em minúsculo.

- [x] Alterado

LuccasArruda commented on src/Core/Entities/Rede_Eletrica.cs:
```diff
@@ -0,0 +1,15 @@
+
+namespace Deméter
+{
+    internal class Rede_Eletrica
```
# ℹ️ Requisição de Mudança
> O padrão de nomenclatura do C# para classes é o ``PascalCase``, ou seja, a primeira letra de cada palavra deve ser em caps-lock e as demais, em minúsculo.

- [x] Alterado

LuccasArruda commented on src/Core/Entities/Rede_Eletrica.cs:
```diff
@@ -0,0 +1,15 @@
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deméter
```
# ℹ️ Requisição de Mudança
> Seria interessante se o namespace do projeto fosse escrito sem acento.

LuccasArruda commented on src/Core/Entities/Usuario.cs:
```diff
@@ -0,0 +1,18 @@
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deméter
```
# ℹ️ Requisição de Mudança
> Seria interessante se o namespace do projeto fosse escrito sem acento.

LuccasArruda commented on src/Core/Entities/Aparelho.cs:
```diff
@@ -0,0 +1,16 @@
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deméter
```
# ℹ️ Requisição de Mudança
> Seria interessante se o namespace do projeto fosse escrito sem acento.

LuccasArruda commented on src/Core/Entities/Ambiente_Profissional.cs:
```diff
@@ -0,0 +1,13 @@
+
+namespace Deméter
+{
+    internal class Ambiente_Profissional: Ambiente
```
# ℹ️ Requisição de Mudança
> O padrão de nomenclatura do C# para classes é o ``PascalCase``, ou seja, a primeira letra de cada palavra deve ser em caps-lock e as demais, em minúsculo.

- [x] Alterado

LuccasArruda commented on src/Core/Entities/Ambiente_Pessoal.cs:
```diff
@@ -0,0 +1,13 @@
+
+namespace Deméter
+{
+    internal class Ambiente_Pessoal: Ambiente
```
# ℹ️ Requisição de Mudança
> O padrão de nomenclatura do C# para classes é o ``PascalCase``, ou seja, a primeira letra de cada palavra deve ser em caps-lock e as demais, em minúsculo.

- [x] Alterado

LuccasArruda commented on src/Core/Entities/Ambiente_Pessoal.cs:
```diff
@@ -0,0 +1,13 @@
+{
+    internal class Ambiente_Pessoal: Ambiente
+    {
+        public Rede_Eletrica RedeEletrica { get; private set; }
```
# ℹ️ Requisição de Mudança
> O padrão de nomenclatura do C# para classes é o ``PascalCase``, ou seja, a primeira letra de cada palavra deve ser em caps-lock e as demais, em minúsculo.

- [x] Alterado

LuccasArruda reviewed (changes requested):
# ⚠️ Parabéns pelo código!
> O código está muito bem feito, porém, algumas coisas poderiam melhorar.

_todas as requisições de mudança estão aberta à discussão_

GabrielBrandi commented on src/Core/Entities/Aparelho.cs:
```diff
@@ -0,0 +1,16 @@
+{
+    internal class Aparelho
+    {
+        public double Watts { get; private set; }
```
# ℹ️  Requisição de Mudança
> Watts é um nome muito genérico, uma propriedade deve receber um nome mais descritivo, para que seu uso não fique confuso, como "potência" ou algo parecido.
- [x]   Alterado

Commit:
refactor(core): renomear classes, namespace e atributo para padronização

- Renomeado namespace de 'Deméter' para 'Demeter'
- Renomeada classe 'Ambiente_Profissional' para 'AmbienteProfissional'
- Renomeada classe 'Ambiente_Pessoal' para 'AmbientePessoal'
- Renomeada classe 'Rede_Eletrica' para 'RedeEletrica'
- Renomeado atributo 'Watts' para 'Potencia'

LuccasArruda reviewed (approved):
# ✅ Aprovado!
> Parabéns pelo código!

Skyblockfire reviewed (approved):
Problemas corrigidos, PR aceito.
